app/src/main/java/com/release/keyneez/presentation/main/explore/recent/RecentFragment.kt

```diff
@@ -0,0 +1,40 @@
+package com.release.keyneez.presentation.main.explore.recent
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.viewModels
+import com.release.keyneez.R
+import com.release.keyneez.databinding.FragmentExploreRecentBinding
+import com.release.keyneez.domain.model.ExploreData
+import com.release.keyneez.util.binding.BindingFragment
+
+class RecentFragment :
+    BindingFragment<FragmentExploreRecentBinding>(R.layout.fragment_explore_recent) {
+
+    private val viewModel: RecentViewModel by viewModels()
+    val data = mutableListOf<ExploreData>()
+    private lateinit var RecentAdapter: RecentAdapter
```

Comment:
```suggestion
    private lateinit var recentAdapter: RecentAdapter
```
그리구 깜빡하신 것 같긴 하지만 상수가 아닌 일반 변수는 lowerCamelCase로 작성하는 것이 [공식문서에서 안내하고 있는 관례](https://developer.android.com/kotlin/style-guide#non-constant_names)입니다!

Reply:
넵 감사합니다! 더 신경 써 볼게요:)